### PR TITLE
Change the way onClickImage is propagated

### LINKED ISSUE #171: Not setting onClickImage produces prop-types warning
After upgrading my project dependencies, I started seeing this warning each time Lightbox gets opened:

```
Warning: Expected `onClick` listener to be a function, instead got a value of `boolean` type.
    in img (created by Lightbox)
    in figure (created by Lightbox)
    in div (created by Lightbox)
    in div (created by Container)
    in Container (created by Lightbox)
    in CSSTransitionGroupChild (created by TransitionGroup)
    in div (created by TransitionGroup)
    in TransitionGroup (created by CSSTransitionGroup)
    in CSSTransitionGroup
    in div
    in PassContext
```

I suspect that it is caused by: [Lightbox.js#L244](https://github.com/jossmac/react-images/blame/243c98ea4efb7ead010668012d51744a85a436aa/src/Lightbox.js#L244).

Adding  a dummy `onClickImage` prop helps:

```js
<Lightbox
  // ...
  onClickImage={() => console.log('test')}
/>
```

However, this also changes a cursor, which is not desired.

**Steps to reproduce the behavior:**

Just use the latest version of react and the latest version of react-images. Do not set `onClickImage` to `<Lightbox />`

### PR DESCRIPTION
**Pull request by kachkaev**
**Related issues (if any):**

Fixes #171

**Checks:**

- [ ] Please confirm `npm run lint` ran successfully

> I edited the file online without cloning. Hope this is fine for such a tiny change.

- [x] Please confirm that only `/src` and `/examples/src` are committed
> Only changed a couple of lines in one file

- [ ] [if new feature] Please confirm that documentation was added to the README.md

> NA

### CODE REVIEW

File: src/Lightbox.js [hunk 1]
```diff
@@ -241,13 +241,13 @@ class Lightbox extends Component {
...
 					className={css(this.classes.image)}
-					onClick={!!onClickImage && onClickImage}
+					onClick={onClickImage}
 					sizes={sizes}
 					alt={image.alt}
 					src={image.src}
 					srcSet={srcset}
 					style={{
-						cursor: this.props.onClickImage ? 'pointer' : 'auto',
+						cursor: onClickImage ? 'pointer' : 'auto',
```
Comment (kachkaev) [inline src/Lightbox.js]: No need for this.props since this has been done earlier in `const {...} = this.props;`

**Comment by kachkaev** [comment]
@neptunian I'm deciding between releasing a version with the warning or waiting for a new release if it happens fast. If you're not too busy, could you give a hint when you can have a look at this? Either way works for me, just curious.

**Comment by neptunian** [comment]
Sorry for late response @kachkaev

**Comment by kachkaev** [comment]
Thanks for merging! Could you please cut a new release on npm? Not sure what the version should be, because the latest ones differ in this repo and on npm.

**Comment by knro** [comment]
As discussed in https://github.com/benhowell/react-grid-gallery/issues/83, it appears that this issue is still present? I have react-images v0.5.17 installed.

**Comment by Markus357** [comment]
Has this issue resurfaced? Currently experiencing it on a small project:
```
"dependencies": {
    "query-string": "^6.2.0",
    "react": "^16.6.3",
    "react-dom": "^16.6.3",
    "react-grid-gallery": "^0.5.3",
    "react-images": "^0.5.19",
    "react-paginate": "^6.0.0",
    "react-scripts": "2.1.1",
    "styled-components": "^4.1.2"
  },
```

**Pull request merged**